modules/migration-mtc-release-notes-1-8.adoc

```diff
@@ -46,6 +46,33 @@ In this release, on migrating an application including a `BuildConfig` from a so
 In this release, after enabling `Require SSL verification` and adding content to the CA bundle file for an MCG NooBaa bucket in MigStorage, the connection fails as expected. However, when reverting these changes by removing the CA bundle content and clearing `Require SSL verification`, the connection still fails. The issue is only resolved by deleting and re-adding the repository. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2240052[*BZ#2240052*])
 
 
+.Backup phase fails after setting custom CA replication repository
+
+In {mtc-full} ({mtc-short}), after editing the replication repository, and adding a custom CA, and the repo is successfully connected, when triggering a migration, there is a failure returned on the backup phase.
+
+This issue has been resolved in in {mtc-short} 1.8.2.
```

Comment:
```suggestion
This issue has been resolved in {mtc-short} 1.8.2.
```
- Remove repeated word
- I think you can delete this line as redundant. You've already mentioned that these are fixed issues, correct?